dipy/workflows/io.py

```diff
@@ -28,6 +29,65 @@
 ne, have_ne, _ = optional_package("numexpr")
 
 
+class StatsPropertyName(enum.Enum):
+    """Statistical data property names."""
+
+    MIN = "min"
+    MAX = "max"
+    MEDIAN = "median"
+    MEAN = "mean"
+    STD_DEV = "std dev"
+
+
+class PercentilePropertyName(enum.Enum):
+    """Percentile data property names."""
+
+    PERCENTILE_2 = "2nd percentile"
+    PERCENTILE_98 = "98th percentile"
+
+
+class VolumetricPropertyName(enum.Enum):
+    """Volumetric data property names."""
+
+    AFFINE = "Affine matrix"
+    VOXEL_ORDER = "Voxel order"
+    DATA_TYPE = "Data type"
+    DIMENSIONS = "Dimensions"
+    VOXEL_SIZE = "Voxel size"
+
+
+class BvalPropertyName(enum.Enum):
+    """b-value data property names."""
+
+    B0_THRESHOLD = "b0 threshold"
+    B_VALUES = "b-values"
+    NUMBER_B0s = "Number of b0s"
+    NUMBER_B_VALUES = "Total number of b-values"
+    NUMBER_SHELLS = "Number of gradient shells"
+
+
+class BvecPropertyName(enum.Enum):
+    """b-vector data property names."""
+
+    B_VECTORS = "b-vectors"
+    B_VECTORS_SHAPE = "Shape of b-vectors on disk"
+    NUMBER_NONUNIT_B_VECTORS = "Total number of non-unit b-vectors"
+    NUMBER_UNIT_B_VECTORS = "Total number of unit b-vectors"
+
+
+class TractographyPropertyName(enum.Enum):
+    """Tractography data property names."""
+
+    DPP_KEYS = "Data per point keys"
+    DPS_KEYS = "Data per streamline keys"
+    LENGTH_MM = "Length (mm)"
+    LENGTH_NUM_PTS = "Length (nb points)"
+    NUM_STRML = "Number of streamlines"
+    ORIGIN = "Origin"
+    SPACE = "Space"
+    STEP_SIZE = "Step size (mm)"
```

Comment:
Can you explain why you need `Enum`.

I usually use enum as a datatype which does not seems to be the case here (look like constante).  I do not see where it will be reuse and I do not see the benefit of it.

Reply:
These can be thoughts as constants, and I do not know whether they will be used elsewhere. They all belong to different categories and enums allow to organize them nicely. Please, propose something else if you still believe there is a better, cleaner way.